templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig

```diff
@@ -4,72 +4,78 @@
- * - primary_document_type: The type of the primary document uploaded.
+ * - paragraph: Full paragraph entity.
+ *   Only method names starting with "get", "has", or "is" and a few common
+ *   methods such as "id", "label", and "bundle" are available. For example:
+ *   - paragraph.getCreatedTime() will return the paragraph creation timestamp.
+ *   - paragraph.id(): The paragraph ID.
+ *   - paragraph.bundle(): The type of the paragraph, for example, "image" or "text".
+ *   - paragraph.getOwnerId(): The user ID of the paragraph author.
+ *   See Drupal\paragraphs\Entity\Paragraph for a full list of public properties
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
  *
- * The primary_document_size and primary_document_type variables are linked.
- * You can't have one without the other because they come from the same File
- * entity.
+ * @see template_preprocess_paragraph()
  *
- * - secondary_document_size: The size of the secondary document uploaded.
- * - secondary_document_file_uri: The link to the secondary document uploaded.
- * - secondary_document_type: The type of the secondary document uploaded.
- *
- * The secondary_document_size and secondary_document_type variables are linked.
- * You can't have one without the other because they come from the same File
- * entity.
+ * @ingroup themeable
  */
 #}
-{% extends "paragraph.html.twig" %}
 
-{% block paragraph %}
+{% set primary_type = paragraph.field_primary_action_type.value %}
 
-  {% set primary_type = paragraph.field_primary_action_type.value %}
-  {% set secondary_type = paragraph.field_secondary_action_type.value %}
+{% set classes = [
+  'cta-block',
+  'box-left',
+  primary_type == 'document' ? 'download' : 'link'
```

Comment:
Indentation.

Reply:
Could it be that the lack of a preceding quote gives the impression that it’s not correctly indented? It should be ok..